Fix Javadoc warnings

### Describe the bug

There are javadoc warnings

### Expected behavior

_No response_

### Minimal reproducible example

_No response_

### Add-on Version

5.0.0

### Vaadin Version

-

### Additional information

```
[WARNING] J:\fc\addons\OrgChartAddon\src\main\java\com\flowingcode\vaadin\addons\orgchart\OrgChart.java:260: warning: Tag @link: reference not found: DragAndDropListener
[WARNING] * Adds a {@link DragAndDropListener} to the component.
[WARNING] ^
[WARNING] J:\fc\addons\OrgChartAddon\src\main\java\com\flowingcode\vaadin\addons\orgchart\OrgChart.java:262: warning: @param argument "dragAndDropListener" is not a parameter name.
[WARNING] * @param dragAndDropListener the listener to be added.

--- a/pom.xml
+++ b/pom.xml
@@ -313,7 +313,7 @@
                         <configuration>
                             <quiet>true</quiet>
                             <doclint>none</doclint>
-                            <failOnWarnings>false</failOnWarnings>
+                            <failOnWarnings>true</failOnWarnings>
                             <links>
                                 <link>https://javadoc.io/doc/com.vaadin/vaadin-platform-javadoc/${vaadin.version}</link>
                             </links>

--- a/src/main/java/com/flowingcode/vaadin/addons/orgchart/OrgChart.java
+++ b/src/main/java/com/flowingcode/vaadin/addons/orgchart/OrgChart.java
@@ -257,9 +257,9 @@ public class OrgChart extends Div {
   }
 
   /**
-   * Adds a {@link DragAndDropListener} to the component.
+   * Adds a {@link DragAndDropEvent} listener to the component.
    *
-   * @param dragAndDropListener the listener to be added.
+   * @param listener the listener to be added.
    */
   public void addDragAndDropListener(ComponentEventListener<DragAndDropEvent> listener) {
     addListener(DragAndDropEvent.class, listener);

